subsys/net/ip/tp.c

```diff
@@ -465,14 +463,10 @@ void _tp_output(sa_family_t af, struct net_if *iface, void *data,
 
 		memcpy(net_buf_add(buf, len), data, len);
 
-		net_pkt_frag_add(pkt, buf);
+		net_pkt_append_buffer(pkt, buf);
```

Comment:
and then here get rid of the for () loop and us net_pkt_writg(pkt, data, len). Done.

Reply:
done